release.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+set -euo pipefail
+
+VERSION="$(date +%Y.%m.%d)"
+LASTEST_VERSION="$(git tag --list | tail -n 1)"
+
+if [ "${VERSION}" = "${LASTEST_VERSION}" ]; then
+    echo "Nothing to release"
+    exit 0
+fi
+
+# Make sure local fork is updated
+git fetch -p --all
+git checkout master
+git merge --ff-only upstream/master
+
+# Bump version number
+echo "${VERSION}" > VERSION
+
+# Generate the package
+make package-clean package
+
+# Sanity check Pulp Smash packages on both Python 2 and Python 3
+for python in python{2,3}; do
+    virtualenv -p ${python} pulpsmash-${python}
+    source pulpsmash-${python}/bin/activate
+    for dist in dist/*; do
+        pip install "${dist}"
+        python -m pulp_smash
+        pip uninstall -y pulp_smash
+    done
+    deactivate
+    rm -rf pulpsmash-"${python}"
+done
+
+# Get the changes from last release and commit
+cat > commit_message <<EOF
+Realease version ${VERSION}
+
+Shortlog of commits since last release:
+
+EOF
+git shortlog "${LASTEST_VERSION}".. | sed "s/^./    &/" >> commit_message
+git commit -a -F commit_message
+rm commit_message
```

Comment:
How about this?

```sh
git commit -m 'Release version 1' \
    -m 'Shortlog of commits since last release:' \
    -m "$(git shortlog | sed 's/^./    &/')"
```